Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,61 @@
 import React from 'react';
+import ButtonContainer from './components/ButtonComponents/ButtonContainer';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import NumberButton from './components/ButtonComponents/NumberButton';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
 import './App.css';
 
+//importing FontAwesome icons to use in throughout the app
+import { faPlus, faDivide, faMinus, faEquals, faTimes} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="calc-container">
```

Comment:
I like your organization - it's like Dustin's left to right
If you want to see a different approach, here is my old code 
https://github.com/VivaCode/React-UI-Components/blob/master/Day-II-Calculator/calculator/src/App.js